controllers/front/ProductController.php

```diff
@@ -498,7 +496,27 @@ public function displayAjaxRefresh(): void
             'id_customization' => $product['id_customization'],
             'product_title' => $this->getTemplateVarPage()['meta']['title'],
             'is_quick_view' => $this->isQuickView(),
-        ]));
+        ];
+
+        $modulesAjaxData = Hook::exec(
+            'actionProductControllerAjaxRefresh',
+            [
+                'ajaxData' => &$ajaxData,
+                'product' => $product,
+            ],
+            null,
+            true
+        );
+
+        if (is_array($modulesAjaxData)) {
+            foreach ($modulesAjaxData as $moduleName => $data) {
+                $ajaxData['modules'][$moduleName] = $data;
+            }
+        }
```

Comment:
I don't understand why you need this? since you pass the data by reference the module hook should modify it directly no?

Reply:
@web-cooking-factory still no answer for this question, can you please give use more details or this PR status won't change